source/includes/api/_layer.md

```diff
@@ -185,163 +340,182 @@ Sorting by nested fields is not supported at the moment.
 > Sorting layers
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/layer?sort=name
+curl -X GET "https://api.resourcewatch.org/v1/layer?sort=name"
 ```
 
 Multiple sorting criteria can be used, separating them by commas.
 
 > Sorting layers by multiple criteria
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/layer?sort=name,slug
+curl -X GET "https://api.resourcewatch.org/v1/layer?sort=name,slug"
 ```
 
 You can specify the sorting order by prepending the criteria with either `-` or `+`. By default, `asc` order is assumed.
 
 > Explicit order of sorting
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/layer?sort=-name,+slug
+curl -X GET "https://api.resourcewatch.org/v1/layer?sort=-name,+slug"
 ```
 
 > Sorting layers by the role of the user who owns the layer
 
 ```shell
-curl -X GET https://api.resourcewatch.org/v1/layer?sort=user.role
+curl -X GET "https://api.resourcewatch.org/v1/layer?sort=user.role"
 ```
 
 ### Include related entities
 
-When loading layer data, you can optionally pass an `includes` query argument to load additional data.
+When fetching layers, you can request additional entities to be loaded. The following entities are available:
```

Comment:
How consistent is the `includes` implementation? We might want to consider breaking that out into its own section at the top and then just referencing it for each resource that supports it.

Reply:
not a lot, so far widgets, layers and datasets support different combinations of includes, so having a central section would potentially make it confusing to the cover what is supported where.